### PR TITLE
Implemented global step for updating a cluster

### PR DESCRIPTION
**Pull request by sbuliarca**

### CODE REVIEW

File: src/com/ft/jenkins/provision/ProvisionConstants.groovy [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.ft.jenkins.provision
+
+/**
+ * There is currently no way to define these constants directly in the ProvisionerUtil script, so putting them in a separate class.
+ */
+final class ProvisionConstants {
+
+  public static final String REPO_URL = 'https://github.com/Financial-Times/k8s-aws-delivery-poc.git'
```
Comment (tamas-molnar) [inline src/com/ft/jenkins/provision/ProvisionConstants.groovy]: Can you add a note to the provisioner readme that it's referenced here? If we ever decide it's not a poc, we'll need to change it here too.

Comment (efinlay24) [reply src/com/ft/jenkins/provision/ProvisionConstants.groovy]: Shall we just do the rename now? It's been irritating me for ages, now seems like a good time to just get on and rename it.

The old URL should continue to redirect anyway from what I understand, so I'd hope it doesn't break things too much.

Comment (tamas-molnar) [reply src/com/ft/jenkins/provision/ProvisionConstants.groovy]: @efinlay24 yeah, why not :+1:

File: src/com/ft/jenkins/slack/SlackUtils.groovy [hunk 2]
```diff
@@ -48,7 +48,7 @@ public void sendEnhancedSlackNotification(String channel, SlackAttachment attach
   echo "Sending attachment to slack: ${attachmentJson}"
   withCredentials([[$class: 'StringBinding', credentialsId: credentialId, variable: 'SLACK_TOKEN']]) {
     String requestBody = "token=${env.SLACK_TOKEN}&attachments=${encodedAttachment}&channel=${encodedChannel}&username=${BOT_USERNAME}"
-    echo "Whole request body: ${requestBody}"
+//    echo "Whole request body: ${requestBody}"
```
Comment (tamas-molnar) [inline src/com/ft/jenkins/slack/SlackUtils.groovy]: Do we still need this for debug purposes or can it be removed?

Comment (sbuliarca) [reply src/com/ft/jenkins/slack/SlackUtils.groovy]: I left it there for debug purposes.

File: vars/updateCluster.groovy [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  setCurrentBuildInfo(fullClusterName, gitBranch)
+
+  ProvisionerUtil provisionerUtil = new ProvisionerUtil()
+
+  catchError {
+    node('docker') {
+      buildProvisionerImage(gitBranch)
+
+      stage('update cluster') {
+        //  todo [sb] create CR for staging & prod
```
Comment (tamas-molnar) [inline vars/updateCluster.groovy]: this still valid?

Comment (sbuliarca) [reply vars/updateCluster.groovy]: Good point, it is already done.

**Pull request merged**